### PR TITLE
[Feature][JsonSplit-api]move of processDefinition, the projectCode instead of projectName

### PR DESCRIPTION
**Pull request by brave-lee**
api module #5498
sub task #5505

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java [hunk 1]
```diff
@@ -135,59 +136,57 @@ public Result createProcessDefinition(@ApiIgnore @RequestAttribute(value = Const
...
      * @param processDefinitionCodes process definition codes
-     * @param targetProjectName target project name
+     * @param targetProjectCode target project code
      * @return copy result code
      */
     @ApiOperation(value = "copy", notes = "COPY_PROCESS_DEFINITION_NOTES")
     @ApiImplicitParams({
             @ApiImplicitParam(name = "processDefinitionCodes", value = "PROCESS_DEFINITION_CODES", required = true, dataType = "String", example = "3,4"),
-            @ApiImplicitParam(name = "targetProjectName", value = "TARGET_PROJECT_NAME", required = true, dataType = "String", example = "ddd")
+            @ApiImplicitParam(name = "targetProjectCode", value = "TARGET_PROJECT_CODE", required = true, dataType = "String", example = "ddd")
```
Comment (wen-hemin) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java]: type is Long

**Review by wen-hemin — COMMENTED** [review]
Good job.
Please check.

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java (hunk 1 above)
Comment (brave-lee) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java]: ths, i will fix it

**Review by wen-hemin — APPROVED** [review]
+1

**Pull request merged**